cob_sound/ros/src/sound.cpp

```diff
@@ -6,6 +6,8 @@
 #include <std_srvs/Trigger.h>
 #include <cob_sound/SayAction.h>
 #include <cob_sound/SayText.h>
+#include <cob_sound/PlayAction.h>
+#include <cob_sound/PlayFile.h>
 
 class SayAction
```

Comment:
this is no longer just a `SayAction`...can we rename it to something like `SoundAction` or `SoundServer`?